LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
```

Comment:
- [ ] To match the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) and the project requirements, kindly ensure that your `comments` table has column for `foreign_key` with corresponding index(for posts and users table). I suggest you add those columns and index in separate migration file. **When creating the reference column you should ensure that your `Comment` belongs to an `:author`** and not to `user`. To help you just refer this **[article](https://stackoverflow.com/questions/27809342/rails-migration-add-reference-to-table-but-different-column-name-for-foreign-ke/42214988)**